bin/fetch-from-genbank

```diff
@@ -62,9 +62,17 @@ params = {
     # Stable sort with newest last so diffs work nicely.  Columns are source
     # data fields, not our output columns.
     'sort': 'SourceDB_s desc, CollectionDate_s asc, id asc',
+
+    # This isn't Entrez, but include the same email parameter it requires just
+    # to be nice.
```

Comment:
Why is this is a nice thing to do? In case someone else wants to run this code but against Entrez?

Reply:
Nice to NCBI, who may notice an `email` parameter in their request logs when they're digging into burdensome requests.